sycl/source/detail/kernel_bundle_impl.hpp

```diff
@@ -380,7 +380,8 @@ class kernel_bundle_impl {
   // program manager integration, only for sycl_jit language
   kernel_bundle_impl(context Ctx, std::vector<device> Devs,
                      const std::vector<kernel_id> &KernelIDs,
-                     std::vector<std::string> KNames, std::string Pfx,
+                     std::vector<std::string> KNames,
+                     sycl_device_binaries Binaries, std::string Pfx,
```

Comment:
What kind of type is `sycl_device_binaries`? How expensive is it to pass it around by value and store it?

Reply:
Cheap; it's just a pointer (to `sycl_device_binaries_struct`).